index.html

```diff
@@ -22,16 +22,44 @@
             </nav>
         </header>
         <section class="hero">
-            <h1>Hector<br>Pereira</h1>
-            <p>Hello! I am a software developer! I can help you build a product, feature or website. Take a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
-            <button>
-                <span>Start collaboration</span>
-            </button>
+            <div class="hero-content">
+                <h1>Hector<br>Pereira</h1>
+                <p>Hello! I am a software developer! I can help you build a product, feature or website. Take a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+                <button>
+                    <span>Start collaboration</span>
+                </button>
+            </div>
         </section>
         <section class="projects">
-            <h2>Projects</h2>
-            <div class="project-box">
-                <img src="img/project-1.png" alt="project-1" class="project-img">
+            <div class="project-title">
+                <h2>Projects</h2>
+            </div>
```

Comment:
Kindly style this section as shown in the original design
![right1](https://user-images.githubusercontent.com/58376020/121605529-daac9200-ca4c-11eb-962e-d5ce41c742bd.png)

Reply:
Please notice that using the vw unit in the width property is causing an overflow on the website. Kindly keep in mind that vw stands for "viewport width" and this means that this element is going to take all the available space on the screen including the document margin. Please use the % unit instead, in order to fix the design issue.